vms/platformvm/state/merkle_state_helpers.go

```diff
@@ -95,15 +95,10 @@ func merkleRewardUtxoIDKey(txID, utxoID ids.ID) []byte {
 	return key
 }
 
-func merkleUtxoIndexPrefix(address []byte) []byte {
```

Comment:
nit: I'd keep the function as we do with other prefixes? I feel it's a better code organization.